app/components/facility/AddFacilityComponent.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useState} from 'react';
+import {Button, Modal} from 'react-bootstrap';
+import {JSONSchema6} from 'json-schema';
+import JsonSchemaForm, {IChangeEvent} from 'react-jsonschema-form';
+import moment from 'moment-timezone';
+import FormFieldTemplate from '../../containers/Forms/FormFieldTemplate';
+import FormObjectFieldTemplate from '../../containers/Forms/FormObjectFieldTemplate';
+
+interface Props {
+  handleAddFacility: any;
```

Comment:
The prop name should be `onAddFacility` The `handleX` convention is for methods inside of the component